strawberryfields/logger.py

```diff
@@ -108,7 +110,23 @@ def create_logger(name, level=logging.INFO):
     no_handlers = not logging_handler_defined(logger)
 
     if effective_level_inherited and level_not_set and no_handlers:
-        logger.setLevel(level)
-        logger.addHandler(default_handler)
+        # The root logger should pass all log message levels
+        # to the handlers.
+        logger.setLevel(logging.DEBUG)
+        level = level or getattr(logging, SESSION_CONFIG["logging"]["level"].upper())
```

Comment:
`SESSION_CONFIG["logging"]["level"]` will result in a `KeyError` if the `level` option is not included in the configuration file or there is no configuration file at all